src/pages/recipes/events/index.md

```diff
@@ -0,0 +1,139 @@
+---
+keywords:
+  - UXP Events
+  - Event Scripting
+---
+# Events
+This section shows how to work with InDesign UXP events.
+
+## Overview
+InDesign can respond to standard application and document events, such as opening a file, creating a new one, printing, and importing text and graphic files from a disk, etc. Scripts can be attached to events using the eventListener scripting object. Scripts that use events are the same as other scripts except that after the event listener is associated, the script is expected to be blocked by a promise and wait for the event in the background until the event listener is removed. Then the promise can be resolved to exit the script. Once you run the Events Scripts with an event listener, they are called back automatically after the corresponding event occurs, rather than being run by the user (from the Scripts palette).
+
+You can see the sample scripts in order of complexity, starting with very simple scripts and building toward more complex operations.
+>
+*Learn how to create, install, and run a script with [Adobe InDesign UXP Scripting Tutorial](https://developer.adobe.com/indesign/uxp/guides/getting-started/).*
+
+## Understanding Event Scripting
+To respond to an event, you register an eventListener with an object capable of receiving the event. When the specified event reaches the object, the eventListener executes the script function defined in its handler function (which can be either a script function or a reference to a script file on disk).
+>
+*Here is the [list of available events](https://developer.adobe.com/indesign/dom/api/e/Event/).*
+
+## Working with Event Listeners
+
+**Adding an eventListener**
+
+When you create an eventListener, you specify the event type and the event handler (as a function or file reference). The following script fragment shows how to add an eventListener for a specific event (for the complete script, see AddEventListener).
+
+```js
+let myInDesign = require("indesign");
+let app = myInDesign.app;
+let myEventListener = app.addEventListener("afterNew", myDisplayEventType);
+```
+The preceding script fragment refers to the following function:
+
+```js
+function myDisplayEventType(myEvent){
+    alert("This event is the " + myEvent.eventType + " event.");
```

Comment:
Use console.log

Reply:
console.log (`This event is the ${myEvent.eventType} event.`);

Reply:
Done